tree/ntuple/v7/src/RPageStorageRaw.cxx

```diff
@@ -258,6 +269,8 @@ ROOT::Experimental::Detail::RPage ROOT::Experimental::Detail::RPageSourceRaw::Po
 
    auto bytesOnStorage = (element->GetBitsOnStorage() * pageInfo.fNElements + 7) / 8;
    if (pageSize != bytesOnStorage) {
+      RNTuplePlainTimer timer(*fCtrTimeWallUnzip, *fCtrTimeCpuUnzip);
```

Comment:
Isn't that (sometimes) a nullptr dereference?

Reply:
The counters would be always there, initialized in the `RPageStorageRaw` constructor and owned by the `Metrics` class.  The counters are of course not necessarily always enabled.

Reply:
I missed that because of the signature of MakeCounter.  Consider replacing 
```
fMetrics.MakeCounter("timeWallRead", "ns", "wall clock time spent reading", fCtrTimeWallRead);
```
with
```
fCtrTimeWallRead = fMetrics.MakeCounter("timeWallRead", "ns", "wall clock time spent reading");
```

Reply:
Yup, signature adjusted.